tyk-pro/scripts/bootstrap.sh

```diff
@@ -22,6 +22,31 @@ syntax_message(){
   exit 1
 }
 
+check_for_existing_organisations(){
+    EXISTING_ORGS=$(CURL_CA_BUNDLE=$CURL_CA_BUNDLE_TYK curl --silent --header "admin-auth: $TYK_ADMIN_SECRET" --header "Content-Type:application/json"  $DASHBOARD_HOSTNAME/admin/organisations 2>&1)
+    OWNER_NAMES=$(echo "$EXISTING_ORGS" | jq -r '.organisations | .[] | .owner_name | @base64')
+    if [[ -z "${OWNER_NAMES}" ]]
+    then
+      echo "No organisation present"
+    else
+      for row in $OWNER_NAMES; do
+        CURRENT_ORG_NAME=$(echo ${row} | base64 -d)
+        if [[ "$CURRENT_ORG_NAME" == "$TYK_ORG_NAME" ]]; then
+           echo "7"
+           return
+        fi
+      done
+      OWNERCNAMES=$(echo "$EXISTING_ORGS" | jq -r '.organisations | .[] | .cname | @base64')
```

Comment:
Unless I am missing some technical detail, we may skip encoding `OWNERCNAMES` because at line 41, we are decoding it again

Reply:
Same as above.

Reply:
Assume you have these two orgs: "Default org1" "Default org2"
If we wouldn't encode and decode it the space between the words "default" and "orgX" would cause our for loop to run 4 times instead of 2 times.